tests/Fractions.Tests/FractionSpecs/Sqrt/Method_Sqrt.cs

```diff
@@ -47,7 +48,7 @@ public void The_result_should_be_equal_to_Math_Sqrt(double value) {
         var expected = Math.Sqrt(value);
         var fraction = Fraction.FromDouble(value);
         // Act
-        var actual = fraction.Sqrt();
+        var actual = fraction.Sqrt(15);
```

Comment:
Note: this test broke as it started actually using the default 30 digits accuracy..